src/persist-client/ci/mzbuild.yml

```diff
@@ -0,0 +1,15 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+name: maelstrom-persist
+pre-image:
+  - type: cargo-build
+    example: persistcli
+    strip: false
+publish: false
```

Comment:
without this, I get an error in the build job. but with it, the mzcompose invocation seems to cargo build the persistcli  example. advice?

https://buildkite.com/materialize/tests/builds/35920#01814fb4-b269-4969-a952-ad5f82521674

Reply:
Ah yeah, we have a funny setup where I have to go into Docker Hub and pre-create the repository for `publish: true`. Fixed now!